akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/CapturedLogEvent.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor.testkit.typed
+
+import java.util.Optional
+
+import akka.actor.typed.LogMarker
+import akka.annotation.InternalApi
+import akka.event.Logging.LogLevel
+import akka.util.OptionVal
+
+import scala.collection.JavaConverters._
+import scala.compat.java8.OptionConverters._
+
+/**
+ * Representation of a Log Event issued by a [[akka.actor.typed.Behavior]]
+ */
+final case class CapturedLogEvent(
```

Comment:
Optional. Add some extra constructors to not pass in errorCause, marker and mdc for the common case of these not being set.

Reply:
I will do later today

Reply:
As it got merged already I will create a new PR with this change :slightly_smiling_face:

Reply:
thanks

Reply:
Created https://github.com/akka/akka/pull/26016